app/views/layouts/application.html.haml

```diff
@@ -22,13 +22,16 @@
                 %button.btn.btn-default{ type: "submit" } Search
         #collapsingContent-lg.collapse.navbar-toggleable-md.pull-lg-right
           %ul.nav.navbar-nav
```

Comment:
The whole list of the menu should probably be extracted into partial or something